wandb/integration/xgboost/xgboost.py

```diff
@@ -87,7 +87,7 @@ class WandbCallback(xgb.callback.TrainingCallback):
         xg_reg.fit(X_train,
             y_train,
             eval_set=[(X_test, y_test)],
-            callbacks=[WandbCallback()])
+            callbacks=[WandbCallback()]
```

Comment:
This change is essentially a bugfix; there's an extra `)` here that prevents `blacken-docs` from correctly parsing this docstring and would cause the GitHub Action to fail if left as-is.